Add security threat submission beta api docs (#16242)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/security-submissionresultdetail.md

| brandImpersonation                     | The reported message was allowed due to the brand impersonation policy settings |
| outboundShouldNotBeBlocked             | The reported outgoing messages has been found clean and Microsoft will update its Machine Learning based Outbound Filters in the coming weeks |
| outboundShouldBeBlocked                | The reported outgoing messages has been found malicious and Microsoft will update its Machine Learning based Outbound Filters in the coming weeks |
| outboundBulk                           | The reported outgoing messages has been found to be bulk and Microsoft will update its Machine Learning based Outbound Filters in the coming weeks |
| outboundCannotMakeDecision             | At the moment Microsoft cannot reach a verdict on the outgoing reported message and so please submit again |
| outboundNotRescanned                   | At the Moment Microsoft cannot reach a verdict as the message cannot be analyzed without taking outside the environment or due to other reasons |
| zeroHourAutoPurgeAllowed               | The reported message was blocked as it was zapped as it was turned malicious after delivery |

> | zeroHourAutoPurgeBlocked               | The Reported message was not zapped despite turning malicious after delivery as the message was bad |

Reviewer: ```suggestion
| zeroHourAutoPurgeBlocked               | The Reported message was not zapped despite turning malicious after delivery as zap was disabled |
```

> | zeroHourAutoPurgeAllowed               | The reported message was blocked as it was zapped as it was turned malicious after delivery |
> | zeroHourAutoPurgeBlocked               | The Reported message was not zapped despite turning malicious after delivery as the message was bad |

Author: ```suggestion
| zeroHourAutoPurgeAllowed               | The reported message was zapped and allowed after delivery as the message was good |
| zeroHourAutoPurgeBlocked               | The Reported message was zapped and blocked after delivery as the message was bad |
```

| invalidFalsePositive                   | The reported message is already allowed                      |
| invalidFalseNegative                   | The reported message is already blocked                      |
| spoofBlocked                           | The reported message is spoof and blocked                    |
| goodReclassifiedAsBad                  | The reported message has already been determined bad by our system |
| goodReclassifiedAsBulk                 | The reported message has already been determined bulk by our system |
| goodReclassifiedAsGood                 | The reported message has already been determined good by our system |
| goodReclassifiedAsCannotMakeDecision   | The reported message does not have a verdict at the moment   |
| badReclassifiedAsGood                  | The reported message has already been identified as good by the system |
| badReclassifiedAsBulk                  | The reported message has already been identified as bulk by the system |
| badReclassifiedAsBad                   | The reported message has already been identified as bad by the system |

> | badReclassifiedAsCannotMakeDecision    | The reported message does not have a decision at the moment  |

Reviewer: ```suggestion
| badReclassifiedAsCannotMakeDecision    | The reported message does not have a verdict at the moment  |
```

> | badReclassifiedAsCannotMakeDecision    |   At the moment Microsoft cannot reach a verdict on the reported message and so please submit again |

Author: ```suggestion
| badReclassifiedAsCannotMakeDecision    | At the moment Microsoft cannot reach a verdict on the reported message and so please submit again |
```
